src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * $cell-size + 6 * $cell-gap + 2 * $calendar-padding;
+  gap: $cell-gap;
+  margin: auto;
+  padding: $calendar-padding;
+  
+  &__day {
+    @include square($cell-size);
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      background-color: pink;
+      cursor: pointer;
+      transform: translateY(-20px);
+      transition: background-color 0.5s, transform 0.5s;
+    }
+  }
+
+  @for $index from 1 through 31 {
+    &__day:nth-child(#{$index})::before {
+      display: block;
+      position: relative;
```

Comment:
why do u need it here?

Reply:
I need it to add numbers in the center of cells. I can't use property top without position.